src/compose/ComposeBox.js

```diff
@@ -88,7 +88,7 @@ export default class ComposeBox extends PureComponent<Props, State> {
     isMessageFocused: false,
     isTopicFocused: false,
     isMenuExpanded: false,
-    height: 40,
+    height: 23,
```

Comment:
How did you come up with the value of 23? It is likely the correct value to be 30?
Considering this line:
```
const messageHeight = Math.min(Math.max(MIN_HEIGHT, height + 12), MAX_HEIGHT);
```

Reply:
Just after render [handleHeightChange](https://github.com/zulip/zulip-mobile/blob/e052865de90103d2922973113e1eb51d2bf9bf6e/src/compose/ComposeBox.js#L119) is called with height as `23`. (From here magic number 23 had came :)) Thus changing height from 42 with 23 and change in height of compose box. 

So now after `handleHeightChange` is called, height is same as before, thus we have also saved one re-render :)

Tested on iPhone 6, 5S, 8 Plus, X and it is working fine 😃

Reply:
Fair enough :)